.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,14 @@ Closes #0000, References #0000, etc.
 - ...
 - ...
 
+
+#### Testing
+
+<!--
+How did you verify that this change works and doesn't cause any regression?
+Check the issue template for expected test scenarios and make sure to add logs/screenshots here that verify those.
```

Comment:
I think the first line is sufficient. I don't think we should refer to the issue template or ask developers to copy-paste log output. I do like the idea of adding screenshots for frontend changes, but IMO those don't belong in the "testing" section.